uctMaths/competition/compadmin.py

```diff
@@ -630,6 +635,33 @@ def export_awards(request):
     output_workbook.save(response)
     return response
 
+def makeCertificates(students, assigned_school):
+    
+    certPath = "C:/Users/dspies/work/uct/git/uct-maths-competition/Certificates/"
+
+    if len(students) > 0:
+        awardCerts = {"G": "goldTemplate.docx", "M": "meritTemplate.docx", None: "participationTemplate.docx", "MOX": "meritOxfordTemplate.docx", "OX": "oxfordTemplate"}
+        schoolname = ''.join([i if ord(i) < 128 else '' for i in repr(assigned_school)])
+        schoolname = schoolname[8:-1].strip()
+        tmpFile = tempfile.NamedTemporaryFile(prefix=str(schoolname))
```

Comment:
This is not right, you don't want a temporary file, you want a temporary directory. Do soemthing like

``` python
tmpDir = tempfile.mkdtemp()
try:
    # <<Generate zip file and send to user>>
finally:
    shutil.rmtree(tmpDir)
```